spec/toArray_spec.js

```diff
@@ -0,0 +1,24 @@
+import { toArray } from '../dist/toArray.js'
+
+describe('toArray', () => {
+  it('converts strings to array', () => {
+    expect (toArray('hi')).toEqual(['h', 'i'])
+  })
+  
+  it('converts objects to array', () => {
```

Comment:
Would be more clear to state that it `converts objects to an array of that object's values`